analysis/result.proto

```diff
@@ -12,12 +12,12 @@ message MethodResultProto {
   string method_name = 1;
 
   repeated NetworkResultProto results_by_network = 2;
-  DistributionProto request_bytes_per_page_view = 3;
-  DistributionProto response_bytes_per_page_view = 4;
 
-  uint64 total_request_count = 5;
-  uint64 total_request_bytes = 6;
-  uint64 total_response_bytes = 7;
+  reserved 3;
+  reserved 4;
+  reserved 5;
+  reserved 6;
+  reserved 7;
```

Comment:
you can use a range: 
```
reserved 3 to 7;
```
It's good practice to also reserve the field names (not strictly necessary in this case):
```
reserved "total_request_count", "total_request_bytes", ...
```